addons/mrp_repair/models/mrp_repair.py

```diff
@@ -225,6 +225,29 @@ def action_repair_cancel(self):
         return self.write({'state': 'cancel'})
 
     @api.multi
+    def action_send_mail(self):
+        self.ensure_one()
+        template_id = self.env.ref('mrp_repair.email_template_send_mrp').id
+        compose_form_id = self.env.ref('mail.email_compose_message_wizard_form').id
+        ctx = dict()
+        ctx.update({
```

Comment:
Why creating a dict in two steps, and updating a void dict with another temporary dict ?